pennylane/templates/subroutines/elbow.py

```diff
@@ -0,0 +1,204 @@
+# Copyright 2025 Xanadu Quantum Technologies Inc.
+
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+
+#     http://www.apache.org/licenses/LICENSE-2.0
+
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+"""
+Contains the Elbow template.
+"""
+
+from functools import lru_cache
+
+import pennylane as qml
+from pennylane.decomposition import add_decomps, register_resources
+from pennylane.operation import Operation
+from pennylane.wires import WiresLike
+
+
+class Elbow(Operation):
```

Comment:
Minor question: why is it added as a template and not as a PL gate?

Reply:
There is no clear answer, in fact at first I put it as Operation and then changed it to template after discussing it with Jay. I think the Elbow is a “strange” operator because in the future it will have MCMs and treating it as a template could give more flexibility

Reply:
This is mostly a marketing/UI point, right? Or are there technical implications of an `Operation` being a template (I don't think there is a consequence in code of this distinction?) 
That being said, I would agree that it is not really clear where to put it. In some sense, it also is just some arithmetic operator, for which we have yet another subdirectory :grin:

Reply:
I think pure operators are more strict with the test and they are tested together normally.
If in the future we are going to needed different checks due to MCMs, could be easier as a template (not 100% sure 😄)